pkg/resources/custom_diffs.go

```diff
@@ -47,3 +47,36 @@ func NestedValueComputedIf(key string, showParam func(client *sdk.Client) (*sdk.
 		return param.Value != valueToString(stateValue[0].(map[string]any)["value"])
 	})
 }
+
+type Property interface {
+	GetName() string
+	GetDefault() string
+}
+
+func BoolComputedIf(key, property string, describe func(client *sdk.Client, id sdk.AccountObjectIdentifier) ([]Property, error)) schema.CustomizeDiffFunc {
```

Comment:
Hmm, let's leave it like this. I think we will change it slightly in a moment after my proposal for warehouses. I think that this logic handles the external change incorrectly (but I see no test for it). Let's leave it for now.

Reply:
Okay.